frontend/public/components/notification-drawer.tsx

```diff
@@ -231,7 +231,7 @@ export const ConnectedNotificationDrawer_: React.FC<ConnectedNotificationDrawerP
     true,
   );
   const [isClusterUpdateExpanded, toggleClusterUpdateExpanded] = React.useState<boolean>(true);
-  const prevDrawerToggleState = usePrevious(isDrawerExpanded);
+  const prevDrawerToggleState = usePrevious(isDrawerExpanded, [isAlertExpanded]);
```

Comment:
@jcaianirh I'm a little unclear how the drawer expanded state depends on the alert expanded state. Can you help me understand?

Reply:
@spadgett this may not be the fix i'm looking for...I've put this on hold. Prior to this fix, usePrevious took no dependencies. Since the change the in usePrev the prevToggleState isn't being updated after the first state change (thinking its like passing an empty dependency array to useEffect). isAlertExpanded is not the right dependency here after thinking more about it. I thought I just needed something to trigger the update of prevstate, but this is updating too often.

Reply:
Looks like this might be fixed by #6002? Can you check?

cc @rawagner